app/routes/organen.js

```diff
@@ -0,0 +1,36 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import RSVP from 'rsvp';
+
+export default class OrganenRoute extends Route {
+  @service currentSession;
+  @service session;
+  @service router;
+  @service store;
+
+  beforeModel(transition) {
+    this.session.requireAuthentication(transition, 'login');
+
+    // TODO we probably want some access rights here.
+    // if (!this.currentSession.canAccessBestuursorganen)
+    //   this.router.transitionTo('index');
+  }
+
+  async model() {
+    const bestuurseenheid = this.currentSession.group;
+    const bestuursorganen = await this.getAllBestuursorganen(
+      bestuurseenheid.get('id')
+    );
+
+    return RSVP.hash({
```

Comment:
not paginating here. i think that is fine and that we don't expect too many bestuursorganen